docs/user/user/pings/submitting-custom-pings.md

```diff
@@ -62,6 +55,17 @@ pings::search.submit(pings::SearchReasonCodes::Performed);
 
 </div>
 
+<div data-lang="Javascript" class="tab">
+
+```js
+import * as pings from "./path/to/generated/files/pings.js";
+
+pings.search.submit("performed");
+```
+</div>
+
+<div data-lang="Firefox Desktop" class="tab"></div>
```

Comment:
Same here, let's file follow-ups for the missing docs.

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1714942